cmd/containeragent/initialize/package_test.go

```diff
@@ -77,6 +77,7 @@ func (*importSuite) TestImports(c *gc.C) {
 		"core/user",
 		"core/watcher",
 		"domain/secret/errors",
+		"domain/model/errors",
```

Comment:
I think it's because `apiserver/errors/errors.go` depends on `domain/model/errors`, some facades used in this package uses apiservererors.RestoreError.